metadata-io/src/main/java/com/linkedin/metadata/timeline/TimelineServiceImpl.java

```diff
@@ -337,11 +334,13 @@ private List<ChangeTransaction> computeDiff(@Nonnull EntityAspect previousValue,
     List<ChangeTransaction> semanticChangeTransactions = new ArrayList<>();
     JsonPatch rawDiff = getRawDiff(previousValue, currentValue);
     for (ChangeCategory element : elementNames) {
-      AspectDiffer differ = _diffFactory.getDiffer(entityType, element, aspectName);
-      if (differ != null) {
+      EntityChangeEventGenerator entityChangeEventGenerator =
+          _entityChangeEventGeneratorFactory.getDiffer(entityType, element, aspectName);
```

Comment:
naming: should this method be called "getGenerator"?

Reply:
Good catch, refactored!